vm/testdata/versioned_constants/custom_versioned_constants_multiple.json

```diff
@@ -0,0 +1,4 @@
+{
+    "0.13.2": "./testdata/versioned_constants/0_13_2.json",
+    "0.13.2.1": "./testdata/versioned_constants/0_13_2.json"
```

Comment:
I think you can recover a version from this PR: https://github.com/NethermindEth/juno/pull/2597/files

Reply:
I just wanted to test, if a JSON file with multiple keys is loading properly. Doesn't really matter what is the value here. Should I still change the duplicate for some other version, as you pointed out